games/j.yaml

```diff
@@ -58,6 +58,29 @@
   url: https://bytonic.de/html/jake2.html
   info: Java port of the Quake2 game engine
 
+- name: JavaPop
+  originals:
+  - Populous
+  - 'Populous II: Trials of the Olympian Gods'
+  type: clone
+  repo: 'https://code.google.com/archive/p/javapop/'
```

Comment:
we don't want to add any games that are in google code as it is a defunct service. Are there other mirrors or continuations where this game is available?